src/CentrifugeRouter.sol

```diff
@@ -43,14 +68,13 @@ contract CentrifugeRouter is Auth, ICentrifugeRouter {
         uint256 lockedRequest = lockedRequests[user][vault];
         require(lockedRequest > 0, "CentrifugeRouter/user-has-no-balance");
         lockedRequests[user][vault] = 0;
-        IERC7540(vault).requestDeposit(lockedRequest, msg.sender, address(this));
+        IERC7540(vault).requestDeposit(lockedRequest, user, address(this));
```

Comment:
nit: Probably we should keep the terminology as it is and use terms such as `owner` , `receiver` and in general variable names used across other contracts.